tasks/create-github-release/create-github-release.yaml

```diff
@@ -40,9 +40,7 @@ spec:
         
         cd "$(workspaces.data.path)/$CONTENT_DIRECTORY"
         set -o pipefail
-        gh release create v$RELEASE_VERSION *.zip *.json *SHA256SUMS --repo $REPOSITORY | tee $(results.url.path)
-        # We will add .sig file when we have the sigining step is done
-        #gh release create v$RELEASE_VERSION *.zip *.json *.sig --repo $REPOSITORY 
+        gh release create v$RELEASE_VERSION *.zip *.json *SHA256SUMS *.sig --repo $REPOSITORY | tee $(results.url.path)
```

Comment:
I wonder if there shouldn't be some checks for the files we're trying to copy. Because if, for example, there is no sig file there: `gh` will still be called, but with literally `*.sig` and then it will fail on a missing file. Which I guess is not the end of the world, but it would be nicer to fail explicitly if something is missing. Just a thought.